packages/react/src/components/CollapsibleNavbarItem/__tests__/__snapshots__/CollapsibleNavbarItem.test.tsx.snap

```diff
@@ -0,0 +1,58 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
We should add license header to snapshot files I guess. (Not manually but through some automated process maybe)

Reply:
we can add an issue for this.